Use case diagram + class diagram created

TomasSmid opened the pull request:
[Closes #7] 

First version of diagrams representing the purpose and possible design
of the system for soccer records.

fprochazka commented:
Já doufal že commitneš zdrojaky toho projektu, aby se s tím dalo dál pracovat, ne obrazky

fprochazka commented:
Další věc: obousměrné vazby jsou obecně architecture smell a často ani nejsou potřeba a dělají se tam "ze zvyku". Vyhl bych se jim, pokud nebudou nezbytně nutné.

TomasSmid commented:
Kouknu na to, rozhodně díky za zpětnou vazbu ;) pokusím se to tedy zasadit do toho DAO, kouknu na ty oboustranné vazby, přejmenuju entity i řídící třídy podle tebou navrhované konvence a konečně sem hodím pak i zdrojový soubor - btw. jsme si nějak nerozuměli, měl jsem za to, že chcete jen výstup, ne zdroják, ale samozřejmě žádný problém - dělám to tedy ve Visual Paradigmu (verze 12.1) - během večera to tu bude

TomasSmid committed:
Use case diagram + class diagram created

First version of diagrams representing the purpose and possible design
of the system for soccer records.

fprochazka commented:
Skvělý, díky!
